conformance/src/main/scala/za/co/absa/enceladus/conformance/interpreter/DynamicInterpreter.scala

```diff
@@ -50,16 +52,35 @@ object DynamicInterpreter {
                 experimentalMappingRule: Boolean,
                 enableControlFramework: Boolean,
                 jobShortName: String = "Conformance"
-               ) (implicit spark: SparkSession, dao: EnceladusDAO, progArgs: CmdConfig): DataFrame = {
+               )(implicit spark: SparkSession, dao: EnceladusDAO, progArgs: CmdConfig): DataFrame = {
+
+    //noinspection TypeAnnotation
+    implicit val interpreterContext = InterpreterContext(inputDf.schema, conformance, experimentalMappingRule,
+      enableControlFramework, jobShortName, spark, dao, progArgs)
+
+    applyCheckpoint(inputDf, "Start")
+
+    val conformedDf: DataFrame = applyConformanceRules(ensureErrorColumnExists(inputDf))
```

Comment:
Do we need the type annotation here? The variable's name is `conformedDf`, `Df` is pretty clearly a `DataFrame`.